h/static/scripts/test/time-test.js

```diff
@@ -50,6 +51,34 @@ describe('time', function () {
   });
 
   describe('.toFuzzyString', function () {
+
+    var originalIntl;
+
+    before(function () {
+
+      // Intl isn't implemented in PhantomJS so we have to mock it in these
+      // tests. The mock version returns the results we'd expect from the real
+      // one for the test values that we use.
+      originalIntl = window.Intl;
```

Comment:
I don't think either 2 or 3 adds much in this case so would go for 1.

But what result are we hoping for from the tests here? If we add an optional `Intl` argument there'll be four possible combinations of the two `Intl` values:

1. The test pass in a mock `Intl` and `window.Intl` is defined (e.g. we're running the tests in Chrome) - it should use the mock

2. The test pass in a mock `Intl` and `window.Intl` is undefined (e.g. we're running the tests in PhantomJS) - it should use the mock

3. The test do not pass in a mock `Intl` and `window.Intl` is defined (e.g. we're running the tests in Chrome) - it will use the real `window.Intl`

4. The test do not pass in a mock `Intl` and `window.Intl` is undefined (e.g. we're running the tests in PhantomJS) - it will fall back on `date.toDateString()`.

3 and 4 are a difference between PhantomJS and Chrome. If I have a "it falls back on date.toDateString() when Intl is undefined" test, that test will pass in PhantomJS but fail in Chrome. Unless I replace the real `window.Intl` with `undefined`, in which case the test will pass in both PhantomJS and Chrome.